glslc/src/main.cc

```diff
@@ -212,23 +205,15 @@ int main(int argc, char** argv) {
               << arg << std::endl;
         }
 
-        // We have to insert a null terminator at the end of name, as we are
-        // using libshaderc's API instead of libshaderc_util's. Libshaderc API
-        // assumes a const char* string, while libshaderc_util assume a
-        // string_piece one. The original implementation doesn't append '\0' but
-        // works fine with libshaderc_util because it has 'end_' property. But
-        // libshaderc api doesn't know that so the macro 'name' will include the
-        // 'value' if we don't insert '\0'.
-
-        macro_names.emplace_back(name_piece.data(), name_length);
-        macro_values.emplace_back(
-            equal_sign_loc == shaderc_util::string_piece::npos ||
+        const string_piece value_piece =
+            equal_sign_loc == string_piece::npos ||
```

Comment:
Please put parentheses around the expression before the ?
Makes it clearer and idiot-proof.